exercises/concept/log-line-parser/.docs/instructions.md

```diff
@@ -25,7 +25,7 @@ LogLineParser.new('[ERROR]: Invalid operation').message
 Any leading or trailing white space should be removed:
 
 ```ruby
-LogLineParser.new('[WARNING]:  Disk almost full\r\n').message
+LogLineParser.new("[WARNING]:  Disk almost full\r\n").message
```

Comment:
Single quotes are fine.

Reply:
Indeed, not only fine, but required, as with double quotes those characters would be interpreted instead of showing as escaped.

Reply:
Never mind, that is exactly the problem, is that they are escaped and not literally what they should be, in order to be stripped.  Good catch!